docker-nginx-loadbalancer/nginx/nginx.conf

```diff
@@ -0,0 +1,19 @@
+user nginx;
+# can handle 1000 concurrent connections
+events {
+    worker_connections   1000;
+}
+# forwards http requests
+http {
+        # http server
+        server {
+              # listens the requests coming on port 8080
+              listen 8080;
+              access_log  off;
```

Comment:
Add this:

```
proxy_request_buffering off;
proxy_buffering off;
```

This will make NGINX not read the response/request into memory which adds latency